datadog_checks_base/datadog_checks/base/stubs/log.py

```diff
@@ -15,6 +15,27 @@ def critical(self, msg, *args, **kwargs):
         raise NotImplementedError('The critical log level is reserved for agent shutdowns.')
 
 
+class CheckLoggingAdapter(logging.LoggerAdapter):
```

Comment:
Should we inherit from `datadog_checks_base/datadog_checks/base/log.py#CheckLoggingAdapter` to reduce duplication ?

Reply:
No since stubs are meant to be isolated